js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -765,6 +772,18 @@ const DropDownEditor = TextBox.inherit({
         this.close();
     },
 
+    _attachPopupButtonInitializedHandler() {
+        const { toolbarItems } = this._options.cache('dropDownOptions');
+
+        if(toolbarItems) {
+            toolbarItems.forEach(item => {
```

Comment:
we can use optional chaining here
```suggestion
            toolbarItems?.forEach(item => {
```

Reply:
[55de9b6a7a3b5202c2a1a4c2ff5ff1f21c696c4c].